# goal
string task
string routine
---
#result
int32 result_code
string message
---